sqlmesh/core/snapshot/definition.py

```diff
@@ -716,6 +730,15 @@ def version_get_or_generate(self) -> str:
         """Helper method to get the version or generate it from the fingerprint."""
         return self.version or self.fingerprint.to_version()
 
+    @property
+    def physical_schema(self) -> str:
+        if self.physical_schema_ is not None:
+            return self.physical_schema_
+        _, schema, _ = parse_model_name(self.name)
+        if schema is None:
+            schema = c.DEFAULT_SCHEMA
+        return f"{schema}__{c.SQLMESH}__physical_layer"
```

Comment:
do you really need do you really need physical_layer

Reply:
trying to make it as distinct as possible. Also indicate its purpose as clearly as possible. Otherwise it looks just like any other environment.

Reply:
i think the SQLMESH prefix is enough, adding physical_layer is so noisy

Reply:
I was thinking about clustering it by user's schema name instead. I think I need another `_` between the user's schema and `sqlmesh` so that the physical schema always comes out on top.